kinappserver/tests/book_and_dont_redeem.py

```diff
@@ -43,7 +43,7 @@ def test_book_and_dont_redeem(self):
                  'desc': 'offer_desc',
                  'image_url': 'image_url',
                  'price': 2,
-                 'address': 'GCORIYYEQP3ANHFT6XHMBY7VB3RH53WB5KZZHGCEXYRWCEJQZUXPGWQM',
+                 'address': 'GCYUCLHLMARYYT5EXJIK2KZJCMRGIKKUCCJKJOAPUBALTBWVXAT4F4OZ',
```

Comment:
@fuzzyami can't you have the address as a constant shared between tests somewhere?